workloads/etcd-perf/common.sh

```diff
@@ -0,0 +1,77 @@
+OPERATOR_REPO=https://github.com/cloud-bulldozer/benchmark-operator.git
+CURL_BODY='{"_source": false, "aggs": {"max-fsync-lat-99th": {"max": {"field": "fio.sync.lat_ns.percentile.99.000000"}}}}'
+
+export TERM=screen-256color
+export METADATA_COLLECTION=${METADATA_COLLECTION:-true}
+export ES_SERVER=${ES_SERVER:-https://search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com}
+export ES_INDEX=${ES_INDEX:-ripsaw-fio-results}
+export TOLERATIONS='[{"key": "node-role.kubernetes.io/master", "effect": "NoSchedule", "operator": "Exists"}]'
+export NODE_SELECTOR='{"node-role.kubernetes.io/master": ""}'
+export LOG_STREAMING=${LOG_STREAMING:-true}
+export CLOUD_NAME=${CLOUD_NAME:-test_cloud}
+export TEST_USER=${TEST_USER:-test_cloud-etcd}
+export FILE_SIZE=${FILE_SIZE:-50MiB}
+export SAMPLES=${SAMPLES:-5}
+export LATENCY_TH=${LATENCY_TH:-10000000}
+
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+log() {
+  echo ${bold}$(date "+%d-%m-%YT%H:%M:%S") ${@}${normal}
+}
+
+deploy_operator() {
+  log "Cloning benchmark-operator from ${OPERATOR_REPO}"
+  rm -rf benchmark-operator
+  git clone ${OPERATOR_REPO} --depth 1
```

Comment:
hi @rsevilla87, can we have something like this here https://github.com/cloud-bulldozer/e2e-benchmarking/pull/122

Reply:
done